api/core.py

```diff
@@ -416,7 +430,11 @@ def _archivestream(self, ticket):
         stream = cStringIO.StringIO()
         with tarfile.open(mode='w|', fileobj=stream) as archive:
             for filepath, arcpath, _ in ticket['target']:
-                yield archive.gettarinfo(filepath, arcpath).tobuf()
+                try:
+                    yield archive.gettarinfo(filepath, arcpath).tobuf()
+                except Exception as e:
```

Comment:
I firmly disagree with this change, and I don't think we can claim to be a platform that features or values data integrity with it enabled. That said, because the consensus disagrees, I'm fine with this landing on aloha (not master), since everything else LGTM.